python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -2023,3 +2023,131 @@ def do_evaluate(cls, *dfs: DataFrame) -> DataFrame:
+                    raise NotImplementedError(f"{compression_type=} is not supported.")
+        elif kind == "Json":
+            # TODO: JSON?
+            pass
+        else:
+            raise NotImplementedError(
+                f"Unhandled sink kind: {kind}"
+            )  # pragma: no cover
+
+    def get_hashable(self) -> Hashable:
+        """
+        Hashable representation of the node.
+
+        The option dictionary is serialised for hashing purposes.
+        """
+        schema_hash = tuple(self.schema.items())
+        return (
+            type(self),
+            schema_hash,
+            self.kind,
+            self.path,
+            json.dumps(self.options),
+        )
+
+    @classmethod
+    def do_evaluate(
+        cls,
+        schema: Schema,
+        kind: str,
+        path: str,
+        options: dict[str, Any],
+        df: DataFrame,
+    ) -> DataFrame:
+        target = plc.io.SinkInfo([path])
+
+        if kind == "Csv":
+            serialize = options["serialize_options"]
+            writer_opts = (
+                plc.io.csv.CsvWriterOptions.builder(target, df.table)
+                .include_header(options["include_header"])
+                .names(df.column_names if options["include_header"] else [])
+                .na_rep(serialize["null"])
+                .line_terminator(serialize["line_terminator"])
+                .inter_column_delimiter(chr(serialize["separator"]))
+                .build()
+            )
+            plc.io.csv.write_csv(writer_opts)
+
+        elif kind == "Parquet":
+            metadata = plc.io.types.TableInputMetadata(df.table)
+            for i, name in enumerate(df.column_names):
+                metadata.column_metadata[i].set_name(name)
+
+            builder = plc.io.parquet.ParquetWriterOptions.builder(target, df.table)
+            if (compression := options["compression"]) != "Uncompressed":
+                compression_type, _ = next(iter(compression.items()))
+                builder.compression(
+                    getattr(plc.io.types.CompressionType, compression_type.upper())
+                )
+
+            if options["data_page_size"] is not None:
+                builder.set_max_page_size_bytes(options["data_page_size"])
+            if options["row_group_size"] is not None:
+                builder.row_group_size_rows(options["row_group_size"])
+
+            plc.io.parquet.write_parquet(builder.metadata(metadata).build())
+
+        elif kind == "Json":
+            metadata = plc.io.TableWithMetadata(
+                df.table, [(col, []) for col in df.column_names]
+            )
+            writer_opts = (
+                plc.io.json.JsonWriterOptions.builder(target, df.table)
+                .lines(val=True)
+                .na_rep("null")
+                .include_nulls(val=True)
+                .metadata(metadata)
+                .build()
+            )
```

Comment:
Match the raw output from polars CPU, when using special characters. Follows up #18508 
```suggestion
            writer_opts = (
                plc.io.json.JsonWriterOptions.builder(target, df.table)
                .lines(val=True)
                .na_rep("null")
                .include_nulls(val=True)
                .metadata(metadata)
                .utf8_escaped(False)
                .build()
            )
```

Reply:
I think this specifying `utf8_escaped` should be included in this PR, as a workaround was needed when including the test in 1e60900169fe26532ec5eea627f78ddad4308855

Reply:
Ok, I checked pandas and it escapes special characters too. See https://github.com/rapidsai/cudf/issues/18483#issuecomment-2822252753. So we made only need this option unset for for cudf-polars, unless pandas wants to do the same thing at some point.

Reply:
Thanks for checking. Agreed that we probably only need this for polars (I doubt pandas is likely to change this in the near future)